propagation/baggage.go

```diff
@@ -29,6 +29,19 @@ func (b Baggage) Inject(ctx context.Context, carrier TextMapCarrier) {
 
 // Extract returns a copy of parent with the baggage from the carrier added.
 func (b Baggage) Extract(parent context.Context, carrier TextMapCarrier) context.Context {
+	multiCarrier, isMultiCarrier := carrier.(MultiTextMapCarrier)
+	if isMultiCarrier {
+		return extractMultiBaggage(parent, multiCarrier)
+	}
```

Comment:
Can we refactor to use "comma ok" Go idiom?

```suggestion
	if multiCarrier, ok := carrier.(MultiTextMapCarrier); ok {
		return extractMultiBaggage(parent, multiCarrier)
	}
```